library/src/main/java/com/devbrackets/android/exomedia/core/exoplayer/ExoMediaPlayer.java

```diff
@@ -528,7 +528,9 @@ protected void sendMessage(int renderType, int messageType, Object message, bool
     }
 
     /**
-     * This was pulled from the <i>Deprecated</i> {@link com.google.android.exoplayer2.ExoPlayerImpl#blockingSendMessages(ExoPlayer.ExoPlayerMessage...)}
```

Comment:
This class is now package protected and cannot be referenced from javadoc.

Reply:
JitPack is failing because of a non-addressable javadoc link?

Reply:
Yes, JitPack runs `./gradlew clean -Pgroup=com.github.brianwernick -Pversion=master-4.1.0-g9f151d9-5 -xtest -xlint build publishToMavenLocal` without further configuration and that fails when javadoc encounters problems.
It is also possible to set [`failOnError`](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.javadoc.Javadoc.html#org.gradle.api.tasks.javadoc.Javadoc:failOnError) to `false` in the Gradle build file to avoid such problems in the future, but I was not sure if that is desired.